opensearch-dashboards/opensearch-build/stackable/patches/3.1.0/0002-set-server.host-to-0.0.0.0-in-default-config.patch

```diff
@@ -0,0 +1,18 @@
+From f960bcb699f59416dde917db4302e178aaca3d3a Mon Sep 17 00:00:00 2001
+From: Benedikt Labrenz <benedikt@labrenz.org>
+Date: Tue, 9 Sep 2025 16:42:53 +0200
+Subject: set server.host to 0.0.0.0 in default config
+
+---
+ config/opensearch_dashboards-3.x.yml | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/config/opensearch_dashboards-3.x.yml b/config/opensearch_dashboards-3.x.yml
+index c6e9c6cc..812d391c 100644
+--- a/config/opensearch_dashboards-3.x.yml
++++ b/config/opensearch_dashboards-3.x.yml
+@@ -231,3 +231,4 @@ opensearch_security.multitenancy.tenants.preferred: [Private, Global]
+ opensearch_security.readonly_mode.roles: [kibana_read_only]
+ # Use this setting if you are running opensearch-dashboards without https
+ opensearch_security.cookie.secure: false
++server.host: '0.0.0.0'
```

Comment:
Isn't the Helm chart responsible for setting this (https://github.com/opensearch-project/helm-charts/blob/opensearch-dashboards-3.1.0/charts/opensearch-dashboards/values.yaml#L147)?